public/app/features/dashboard-scene/scene/DashboardControls.tsx

```diff
@@ -1,41 +1,81 @@
+import { css } from '@emotion/css';
 import React from 'react';
 
-import { SceneObjectState, SceneObject, SceneObjectBase, SceneComponentProps } from '@grafana/scenes';
-import { Box, Stack } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import {
+  SceneObjectState,
+  SceneObject,
+  SceneObjectBase,
+  SceneComponentProps,
+  SceneTimePicker,
+  SceneRefreshPicker,
+  SceneDebugger,
+} from '@grafana/scenes';
+import { Box, Stack, useStyles2 } from '@grafana/ui';
+
+import { getDashboardSceneFor } from '../utils/utils';
 
 import { DashboardLinksControls } from './DashboardLinksControls';
 
 interface DashboardControlsState extends SceneObjectState {
   variableControls: SceneObject[];
-  timeControls: SceneObject[];
-  linkControls: DashboardLinksControls;
+  timePicker: SceneTimePicker;
+  refreshPicker: SceneRefreshPicker;
```

Comment:
think this is a good move, the structire is what it is, we don't need an array of arbitrary scene objects 👍